docs/user/model_setup.md

```diff
@@ -0,0 +1,133 @@
+# How to develop an aiXplain hosted model?
```

Comment:
Remove this file from here and add it to the model-interfaces sub-package.
Likewise, link the on-boarding documentation to model-interfaces.